### PR TITLE
Add Live Oak Bank

### PR DESCRIPTION
**Pull request by georgewerbacher**

### CODE REVIEW

File: entries/l/liveoakbank.com.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "Live Oak Bank": {
+      "domain": "liveoakbank.com",
+      "img": "liveoakbank.com.svg",
```
Comment (Carlgo11) [inline entries/l/liveoakbank.com.json:4]: This line isn't needed.
```suggestion
```

File: entries/l/liveoakbank.com.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      "tfa": [
+        "sms",
+        "call",
+        "totp",
+        "custom-software"
+      ],
```
Comment (Carlgo11) [inline entries/l/liveoakbank.com.json:10]: I see that you haven't added any documentation link. Could you find one that describes how to enable 2FA on the site?

If no documentation is available or you can't find it, please attach screenshots of the 2FA setup and/or login process.
> **Note** Remember to blur or crop out any personal information linked to you.

You can attach the screenshots here in a comment like this:
screenshot

File: img/l/liveoakbank.com.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...3-5.9-9.5-12.7-12.4-20.2-2.8-7.4-4.2-15.3-4.2-23.3 0-17.2 6.9-33.7 19-45.9 12.2-12.2 28.7-19 45.9-19s33.7 6.8 45.9 19c12.2 12.2 19 28.7 19 45.9.1 4.8-.4 9.6-1.4 14.3-1 4.7-2.6 9.2-4.6 13.6-2.1 4.3-4.6 8.4-7.6 12.1-3 3.8-6.4 7.2-10.2 10.2 5.9.5 11.7 1.9 17.2 4s10.8 5 15.5 8.5c4.8 3.6 9 7.7 12.7 12.4 3.6 4.7 6.6 9.8 8.8 15.3q4.5-3.9 9.6-6.9 5.2-3 10.7-5.1 5.6-2.1 11.4-3.2 5.9-1.1 11.8-1.2c24 0 46 13.2 57.3 34.3 11.3 21.1 10 46.8-3.3 66.7-13.4 19.9-36.6 30.8-60.4 28.4-23.8-2.4-44.4-17.7-53.5-39.8q-4.5 3.8-9.7 6.9-5.1 3-10.6 5.1-5.6 2.1-11.5 3.2-5.8 1.1-11.8 1.2c-4.2 0-8.5-.4-12.7-1.3-4.3-.8-8.4-2.1-12.3-3.8-4-1.7-7.7-3.8-11.2-6.2-3.6-2.5-6.8-5.3-9.8-8.4q-7.1 5.6-13.9 11.6-6.8 6.1-13.3 12.4-6.5 6.4-12.6 13.1-6.1 6.8-11.9 13.8c-1.4 1.5-2.5 3.3-3.2 5.2-.8 1.9-1.1 4-1 6.1.1 2 .5 4.1 1.4 5.9.9 1.9 2.1 3.6 3.6 5q5.5 4.9 11.2 9.6 5.6 4.6 11.5 9 5.8 4.5 11.9 8.6 6 4.2 12.2 8.1 8.1 4.5 16.6 8.2 8.4 3.6 17.2 6.4 8.8 2.8 17.8 4.7 9.1 1.9 18.3 2.9c3-10.5 8.6-20.1 16.3-27.8 7.7-7.8 17.3-13.5 27.8-16.5s21.6-3.3 32.3-.9c10.7 2.4 20.5 7.5 28.7 14.8.2-8.3 2-16.5 5.3-24.1 3.3-7.6 8-14.5 13.9-20.3 6-5.7 13-10.3 20.7-13.4 7.7-3.1 15.9-4.7 24.2-4.7 36.1 0 57.5 28.8 57.4 64.9z" style="fill:#1ecd7d"/></svg>
```
Comment (Carlgo11) [inline img/l/liveoakbank.com.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1587 1184"><path fill="#1ecd7d" d="M980.52.29c-33.47 0-60.6 27.04-60.6 60.4.1 10.18 1.68 18.74 6.7 27.61-18.1 4.1-39.12 13.9-67.92 38.6-3.67 2.93-7.37 5.8-11.1 8.6-3.8 2.8-7.67 5.5-11.6 8.1-3.87 2.6-7.83 5.1-11.9 7.5-4 2.47-8.07 4.8-12.2 7-.53.33-1.03.67-2.92 1.68l-3.68 1.62c-4.9 2.5-9 2.5-11.5 0-.67-.87-1.23-1.8-1.7-2.8-.4-1-.73-2.03.05-11.25s.86-10.4.86-11.54l-7.91-2.82a64.5 64.5 0 0 0 18.6 3.7c35.72 0 64.68-29.04 64.68-64.88 0-35.83-28.96-64.88-64.68-64.88-35.72 0-64.68 29.05-64.68 64.88a64.9 64.9 0 0 0 36.86 57.9l-3.38-1.21c0 .8-.8 2.5-.8 3.3-5.8 21.4-9.1 36.2-27.1 46-4.2 2.5-8.3 5-13.2 8.2a108.5 108.5 0 0 1-5.75 3.92c-3.22 2.08-4.52 2.88-10.25 5.26-5.73 2.39-6.67 2.35-7.6 2.22-.93-.13-1.83-.37-2.7-.7-.8-.33-1.6-.73-4.12-3.23-2.51-2.5-2.95-3.3-3.28-4.17-4.9-13.9-11.5-31.2-17.2-47.6l-21.4 4.1h4.1c9.1 36.1 18.1 78.9 12.3 83.8a287 287 0 0 1-4.7 3.2l-4.8 3a183.49 183.49 0 0 1-10.2 5.3c-18.9 9.8-45.1 23-64 45.2-23 26.3-32.9 60-38.6 78-1.7 4.1-2.5 7.4-3.3 9.9-5.8 16.4-23.8 50.1-33.7 53.4-2.47-.79-8.88-10.2-20.97-78.27a66 66 0 0 0 12.9-37.03 66 66 0 0 0-8.14-31.5 66 66 0 0 0 23.61 4.7 66 66 0 0 0 0-132 66 66 0 0 0-62.88 50.26c-3.47-3.22-6.98-6.4-10.52-9.56a496.93 496.93 0 0 0-11.1-9.6 700.64 700.64 0 0 0-23.1-18.3l-.25.76a66 66 0 0 0 2.7-13.57 66 66 0 1 0-66 66 66 66 0 0 0 57.77-35.95l-.02.06s6.77 7.07 9.9 10.8c3.13 3.73 6.1 7.6 8.9 11.6 2.87 4 5.53 8.1 8 12.3 2.47 4.2 4.77 8.5 6.9 12.9l1.6 4c.47 1.4.9 2.77 1.3 4.1a97.4 97.4 0 0 1 1.81 7.51 66 66 0 0 0-6.5-.5 66 66 0 0 0 0 132 66 66 0 0 0 19.97-3.97c.49 4.56.9 9.14 1.22 13.76.4 4.93.7 9.87.9 14.8.2 4.93.33 9.87.4 14.8 0 4.93-.07 9.87-.2 14.8-9.75 3.28-163.57-24.19-165.1-24.46a66 66 0 0 0-65.98-65.6 66 66 0 0 0-62.95 50.67 66 66 0 0 0-53.07-26.82 66 66 0 1 0 0 132.01 66 66 0 0 0 63.27-52.27 66 66 0 0 0 52.75 28.43 66 66 0 0 0 60.34-39.8l-.06.34s12.37 3.93 18.7 5.6c6.27 1.67 12.6 3.1 19 4.3 6.4 1.27 12.8 2.3 19.2 3.1 6.47.8 12.97 1.4 19.5 1.8 23.8 2.5 101 20.5 103.4 36.2 1 4.33 2.07 8.63 3.2 12.9 1.13 4.33 2.37 8.6 5.82 19.15 3.45 10.55 5.01 14.72 6.68 18.85l.8 2.5c5.7 15.6 12.3 42.7 8.2 50.1a66.84 66.84 0 0 1-8 7 65.56 65.56 0 0 1-18.7 10c-3.33 1.2-6.77 2.1-15.35 2.68-5.34.37-5.56.43-7.06.57A66 66 0 1 0 336.3 563.1a66 66 0 0 0 33.23 55.96l-.33.04c-3 0-7.41.6-11.27.73a66 66 0 0 0-41.63-15.04 66 66 0 0 0-45.4 18.48c-14.08.12-32.35.6-34.87.68a66 66 0 0 0-63.72-49.55 66 66 0 0 0-44.2 17.67 66 66 0 0 0 1.99-9.97 66 66 0 1 0-66 66 66 66 0 0 0 44.18-17.67 66 66 0 0 0-1.98 9.97 66 66 0 0 0 66 66 66 66 0 0 0 64.48-58.32c1.8.32 8.78 1.51 12.42 2.22 1.91 0 3.81.4 5.05.61a66 66 0 0 0-3.96 19.9 66 66 0 0 0 132.01 0 66 66 0 0 0-5.33-25.48c.53-.07 6.3-.77 10.23-1.63 55.9-5.7 158.5-16.4 181.5 1.7l.8.8C678 716 711.6 745.6 718.2 771.1c9.9 35.3 32 181.6 5.8 308.1-8.2 40.2-41.1 64.9-65.7 91.2a6.96 6.96 0 0 0-2 4c-.3 1.5-.1 3 .5 4.4a9.9 9.9 0 0 0 2.9 3.5c1.3.8 2.8 1.2 4.3 1.2h275.2c1.6 0 3.2-.5 4.5-1.4 1.4-.9 2.4-2.3 2.9-3.8s.6-3.2.1-4.8c-.4-1.5-1.3-2.9-2.6-3.9-32.9-28.8-73.9-89.6-71.5-235 0-16.4 0-35.3-.8-50.1v-19.8a168.3 168.3 0 0 1-.1-15.4c.27-5.13.73-10.27 1.4-15.4.67-5.07 1.6-10.1 2.8-15.1 1.13-5 2.5-9.97 4.1-14.9 17.3-46 107.6-147.9 166.8-187.3 3.3-2.5 7.4-4.9 14.48-8.7 7.09-3.8 8.19-4.13 9.32-4.4 29.6-14 50.1-21.4 86.2-14.8l.47.2c-20.9 11.24-34.13 33.2-34.52 57.3 0 36.45 28.95 66 64.67 66s64.68-29.55 64.68-66a67.27 67.27 0 0 0-2.18-16.23c24.58 13.83 48.28 26.14 70.38 27.73 9 .8 19.7.8 30.4.8.5 8.1 2.6 15.9 6.1 23.2 3.4 7.3 8.2 13.9 14.1 19.4 5.9 5.5 12.8 9.8 20.3 12.8a63.44 63.44 0 0 0 47.7-.1c7.7-3.1 14.7-7.7 20.6-13.4 6-5.8 10.7-12.7 14-20.3 3.3-7.6 5.1-15.8 5.3-24.1 1.44 0 2.69-.18 4.04-.26 11.36 12.04 26.62 18.81 42.52 18.87 33.71 0 61.04-29.55 61.04-66 0-36.46-27.33-66.01-61.04-66.01-32.24.32-58.68 27.7-60.4 62.5h-.06c-8.2-7.3-18-12.4-28.7-14.8a65.42 65.42 0 0 0-32.3.9c-10.5 3-20.1 8.7-27.8 16.5a64.3 64.3 0 0 0-16.3 27.8 184.08 184.08 0 0 1-36.1-7.6c-5.87-1.87-11.6-4-17.2-6.4-5.67-2.47-11.2-5.2-16.6-8.2-4.13-2.6-8.2-5.3-12.2-8.1-4.07-2.73-8.03-5.6-11.9-8.6a311.44 311.44 0 0 1-11.5-9 385.9 385.9 0 0 1-11.2-9.6c-1.5-1.4-2.7-3.1-3.6-5-.9-1.8-1.3-3.9-1.4-5.9a14 14 0 0 1 1-6.1c.7-1.9 1.8-3.7 3.2-5.2 3.87-4.67 7.83-9.27 11.9-13.8a333 333 0 0 1 12.6-13.1c4.33-4.2 13.3-12.4 13.3-12.4l13.38-11.16a66 66 0 0 0 46.83 19.56 66 66 0 0 0 41.98-16.2 66 66 0 0 0 61.21 41.9 66 66 0 0 0 0-132 66 66 0 0 0-41.96 16.19 66 66 0 0 0-52.03-41.17 66 66 0 0 0 21.2-47.91 66 66 0 1 0-132.01 0 66 66 0 0 0 58.07 64.42 66 66 0 0 0-22.47 48.77 66 66 0 0 0 14.35 39.22l-6.95-8.82s-7.3 3.33-10.9 5.2c-3.53 1.8-7.07 3.7-19.83 11.82l-.33.2c-8.4 4.86-14.13 9.4-28.02 23.13-14.02 13.85-28.82 29.45-41.12 30.25-.93.07-1.9.1-6.7-.75l-2.06-.39a66 66 0 0 0 42.17-61.26 66 66 0 0 0-132.01 0 66 66 0 0 0 22.1 48.52 86.2 86.2 0 0 0-16.3 3.58 94 94 0 0 0-8 2.8c-2.02.78-4.47 1.96-5.92 2.65a66 66 0 0 0 .89-4.44 66 66 0 1 0-132.01 0 66 66 0 0 0 40.56 60.86c-5.59 3.83-71.17 48.74-105.02 67.63-2.4 1.6-8.2 4.1-11.5.8-3.3-2.5-4.9-9.9-1.6-19.7 2.33-6 4.77-11.93 7.3-17.8a519.8 519.8 0 0 1 7.7-17.6 520.73 520.73 0 0 1 8.3-17.4c2.8-5.73 5.7-11.43 8.7-17.1 35.3-66.5 80.5-181.6 87.5-199.65 7-18.05 16.8-43.55 51.3-42.75 24.7.9 51.8 0 73.9-14.8l-3.63-4a63.96 63.96 0 0 0 41.73 16.6c35.73 0 64.68-29.55 64.68-66s-28.96-66-64.68-66c-35.72 0-64.67 29.55-64.67 66a66.8 66.8 0 0 0 15 40.65l-5.22-5.74a107.6 107.6 0 0 1-17.2 6.29 98.06 98.06 0 0 1-16.91 2.9c-5.6.4-11.3.3-17-.4a88.4 88.4 0 0 1-16.7-3.4c-4.9-1.6-6.6-2.5-4.9-25.5.8-9 1.6-24.6 2.5-41.9l-4.21-.36c33.72-2.27 60.04-30.65 60.41-65.14 0-36.45-28.96-66-64.68-66-35.72 0-64.68 29.55-64.68 66 .6 31.84 23.39 58.7 54.18 63.87l-9.02-.77a668 668 0 0 1-7.2 21c-2.47 7-5.03 13.97-7.7 20.9a886.35 886.35 0 0 1-16.11 38.72 66 66 0 0 0-54.08-28.32 66 66 0 1 0 0 132 66 66 0 0 0 8.81-1.75c-10.69 23.46-20.68 44.9-25.52 53.25l-.22.45a66 66 0 0 0-56.68-32.57 66 66 0 0 0 0 132 66 66 0 0 0 8.2-1.63l-7.9 15.95c-11.5 21.3-41.9 77.2-74.8 78.8-13.7.78-46.26-11.87-81.59-31.64a66 66 0 0 0 62.28-65.26 66 66 0 0 0-132 0 66 66 0 0 0 4.57 23.02c-2.85-2.16-5.71-4.31-8.46-6.62a164.9 164.9 0 0 1-10.8-9.7c-3.53-3.33-6.93-6.8-10.2-10.4a212 212 0 0 1-9.4-11.1c-4.1-5.8-9.1-11.5-16-20.3s-10.8-15.6-13.7-22.9c-2.9-7.2-4.8-14.8-5.5-22.6a81.58 81.58 0 0 1 8.9-45.1c2.15-5.2 4.98-9.59 7.35-13.99a66 66 0 0 0 61.65 44.1 66 66 0 1 0 0-132.01 66 66 0 0 0-27.36 6.43l.26-.63c10.7-28.8 23.8-55.9 46.8-73.1 9.8-7.4 26.3-17.3 42.7-27.2l.4-.22a67.19 67.19 0 0 0-3.88 19.92c0 36.45 28.95 66 64.67 66s64.68-29.55 64.68-66c-.04-29-18.61-54.57-45.78-63.04 9-4.3 17.93-8.73 26.71-13.36a722.51 722.51 0 0 0 28.9-16.1A757.2 757.2 0 0 0 846 170c9.33-5.93 18.5-12.1 27.5-18.5a104.62 104.62 0 0 0 6.3-5.1l3.1-2.7c1-.93 1.97-1.87 2.9-2.8 3.87-3.8 7.87-7.4 12-10.8 4.2-3.33 8.53-6.5 13-9.5 4.47-2.93 9.07-5.67 13.8-8.2 4.8-2.53 11.31-4.83 13.78-5.87 0 0 21.88 14.24 42.14 14.57 33.46 0 60.6-27.05 60.6-60.4 0-33.37-27.13-60.42-60.6-60.41zM658.07 18.34a63.69 63.69 0 0 0-63.65 63.73c0 35.2 28.5 63.73 63.65 63.73a63.69 63.69 0 0 0 63.64-63.73c0-35.2-28.5-63.73-63.64-63.73z"/></svg>
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Thanks for the pull request @GeorgeWerbacher!
I've attached a few things that need changing.  🙂

**Comment by RedRubble** [comment]
According to their [website](https://www.liveoakbank.com/privacy-security/):
_"Multi-factor authentication when additional verification is needed (asking you a question that only you know the answer, or sending a security code to a device that you’ve registered)."_

So this might be another case of being authorisation, rather than 2FA.

**Comment by Carlgo11** [comment]
Closing due to inactivity.
> [!Note]
> The PR author you can still commit the requested changes and re-open this PR.
> If someone else wants to commit the requested changes you can create a new PR and link to this PR.

**Pull request closed without merging**